docs/sources/oncall-api-reference/schedules.md

```diff
@@ -195,3 +195,159 @@ curl "{{API_URL}}/api/v1/schedules/SBM7DV7BKFUYU/" \
 **HTTP request**
 
 `DELETE {{API_URL}}/api/v1/schedules/<SCHEDULE_ID>/`
+
+# Export a schedule's final shifts
+
+**HTTP request**
+
+```shell
+curl "{{API_URL}}/api/v1/schedules/SBM7DV7BKFUYU/final_shifts?start_date=2023-01-01&end_date=2023-02-01" \
+  --request GET \
+  --header "Authorization: meowmeowmeow"
+```
+
+The above command returns JSON structured in the following way:
+
+```json
+{
+  "count": 12,
+  "next": null,
+  "previous": null,
+  "results": [
+    {
+      "user_pk": "UC2CHRT5SD34X",
+      "user_email": "spongebob@grafana.com",
```

Comment:
Maybe it would be a good idea to use something more neutral like `alice@example.com`, `bob@example.com`, etc.? 
I personally like SpongeBob SquarePants a lot, but probably there are people that don't feel the same way 😄